### ISSUE TITLE
documentation: how to change your UHK

### ISSUE THREAD

**Issue by jzacsh** [UltimateHackingKeyboard/agent#674]
Hi, so this is really a documentation bug;

first: the uhk website/docs/FAQs should say something about how to modify your UHK (eg: point to this repo, anything)

second: the README for this repo could really benefit from an expansion on "download this snap file from our releases page"

i'd be more constructive on this second point (or just write a pull request to the REAMDE.md, right now) but I still haven't figured out the installation instructions *myself*. currently i'm stuck at:
- downloaded .snap file
- `xdg-open UHK*.snap` # no good
- `sudo apt install snapd && snap run ./UKH*.snap` # no good
- now [reading part of a forum post](https://forum.snapcraft.io/t/install-snap-files-offline/302/6) saying _to_ but _not_ to do `snap install --dangerous ./UHK*.snap && snap run ./UHK*.snap` and before running that i decided: if this is really the solution, then the doc for this repo should *definitely* say so..

So what's the correct installation of Agent (even if beta/alpha/canary state ..)? Happy to update the README.md with the results of this bug.

**Comment by mondalaci** [comment]
Hi there!

* AppImage is recommended over snaps. We don't directly support snaps, but electron builder builds them regardless.
* We welcome contributors, but users are not expected to pull repos, or build anything. The UHK is a fully-fledged product which seamlessly works out of the box. Users are expected to download Agent and run it.

Have you received your UHK and completed the onboarding guide? I'm asking because it addresses your questions and contains everything needed.

I'm closing this issue because given our onboarding guide, users should be good to go. You're welcome to follow up regardless.

**Comment by jzacsh** [comment]
**tl;dr** for future searcher: http://uhk.io/start

![photo of inside of box 20180612_223922](https://user-images.githubusercontent.com/156228/44241058-277d9a80-a187-11e8-98f2-b44141a5ade1.jpg)

:D so the `uhk.io/start` was accidentally hidden, because when i opened the box, it looked like that message was actually *printed* on the inside of that black space (like a square of white on the cardboard, with some text printed on it), rather than a card stuck there by static.

_(only remembered to finally get back here and reply because I just setup *again* and couldn't remember what that magic URL was)_
